### PR TITLE
Add KeepMacAddress checkbox to Backup restore page

### PR DESCRIPTION
**Pull request by torchiaf**
#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:

Fixes https://github.com/harvester/harvester/issues/5205

### Technical notes summary

Adding a new field to keep the Mac Address from old VM backup.

### Screenshot/Video

Restore/Create new

image

Restore/Replace Existent

image

### CODE REVIEW

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -782,6 +782,7 @@ harvester:
       createNew: Create new
       replaceExisting: Replace existing
       virtualMachineName: Virtual Machine Name
+      keepMacAddress: Keep Mac Address
```
Comment (votdev) [inline pkg/harvester/l10n/en-us.yaml:785]:
```suggestion
      keepMacAddress: Keep MAC Address
```

**Review by FrankYang0529 — APPROVED** [review]
For functionality, LGTM. Thanks.

**Review by Yu-Jack — APPROVED** [review]
LGTM, thanks

**Review by votdev — APPROVED** [review]
LGTM

**Comment by bk201** [comment]
@torchiaf Do we need to backport this issue? cc @albinsun

**Comment by torchiaf** [comment]
> @torchiaf Do we need to backport this issue? cc @albinsun

In theory it should be an enhancement and it has to be relase in v1.3.0-rc4. There is a backport PR for the backend changes, but it's for 1.3 : https://github.com/harvester/harvester/pull/4957#issuecomment-1932994770

**Pull request merged**